ext/wkhtmltopdf-binary/Rakefile

```
@@ -25,6 +25,9 @@ suffix = case RbConfig::CONFIG['host_os']
            os_based_on_ubuntu_20_04 = os.start_with?('ubuntu_20.')
            os = 'ubuntu_20.04' if os_based_on_ubuntu_20_04
 
+           os_based_on_ubuntu_22_04 = os.start_with?('ubuntu_22.') || os.start_with?('ubuntu_24.')
```

```suggestion
           os_based_on_ubuntu_22_04 = os.start_with?('ubuntu_22.')
```
Why did we merge the condition here? Can we create a new line?

---

This matches the upstream logic, which uses the 22 binary for either 22 or 24. Refer: https://github.com/zakird/wkhtmltopdf_binary_gem/blob/master/bin/wkhtmltopdf#L45-L50